lib/ddtrace/monkey.rb

```diff
@@ -48,7 +50,7 @@ def patch(modules)
 
     def get_patched_modules
       patched = autopatch_modules
-      @autopatch_modules.each do |k, v|
+      @patchers.each do |k, v|
```

Comment:
This was needed because a module can be autopatch false, we want to check from the list of all available patchers if they have been patched.